roles/confluent.common/tasks/redhat.yml

```diff
@@ -8,6 +8,7 @@
     gpgcheck: "{{confluent_common_repository_redhat_dist_gpgcheck}}"
     gpgkey: "{{confluent_common_repository_redhat_dist_gpgkey}}"
     enabled: "{{confluent_common_repository_redhat_dist_enabled}}"
+    proxy: "{{proxy_env.http_proxy}}"
```

Comment:
```suggestion
    proxy: "{{proxy_env.http_proxy | default(omit)}}"
```

Reply:
can you test with `omit`? this is useful when the var is not defined

Reply:
> One more suggestion @elfelli but looks good. Can you update the description to show the contents of the repo file with the `proxy_env.http_proxy` env var set and not set?

done !

Reply:
line 11 is missing the default omit

also i meant to ssh to a host and `cat /etc/yum.repos.d/confluent.repo`
and put that into the description, not copy the file changes